.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
```

Comment:
Yes loops should be done using map or filter, but in situations that require a simple increment by 1, i++ is much cleaner. Also it's slightly faster https://jsperf.com/for-loop-i-vs-i/4